main.go

```diff
@@ -136,20 +116,6 @@ func configureHealthChecks(ctx context.Context,
 		hasErrors = true
 	}
 
-	if cfg.HasPrivateEndpoints {
-		if err = hc.AddCheck("Kafka Audit Producer", auditProducer.Checker); err != nil {
-			log.Event(ctx, "error adding check for kafka audit producer", log.ERROR, log.Error(err))
-			hasErrors = true
-		}
-
-		// zebedee is used only for identity checking
-		zebedeeClient := zebedee.New(cfg.AuthAPIURL)
```

Comment:
The Zebedee health check has been removed - is that right? The config is still there and I can't see why it would be removed.